addons/languages/icelandic/apk/src/main/AndroidManifest.xml

```diff
@@ -0,0 +1,7 @@
+<manifest xmlns:android="http://schemas.android.com/apk/res/android"
+          package="com.anysoftkeyboard.languagepack.german">
```

Comment:
Should be com.anysoftkeyboard.languagepack.icelandic

Reply:
right, c&p issue

Reply:
done